### PR TITLE
Add bandchain.js

### PR DESCRIPTION
**Pull request by smiled0g**
Note: the old `bandchain.js` branch got messed up due to an incorrect rebase. This PR is for the new branch with the same content.

### CODE REVIEW

**Review by evilpeach — COMMENTED** [review]
Should we add the test to GitHub action?

**Comment by smiled0g** [comment]
> Should we add the test to GitHub action?

Hmm.. I'm not entirely sure here. I guess we should, but in that case, we should also use mock server requests instead of the real requests to GuanYu devnet we're doing right now.

What do you think? @sorawit

**Comment by sorawit** [comment]
Yes. I think if we are to add it to github action, better be tests that don't rely on external API calls.

**Review by evilpeach — APPROVED** [review]

**Review by prin-r — APPROVED** [review]

File: helpers/bandchain.js/src/lib/BandChain.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    )
+
+    let signedTx = cosmos.sign(requestMsg, ecpairPriv, 'block')
+    convertSignedMsg(signedTx)
+
+    const broadcastResponse = await cosmos.broadcast(signedTx)
+    return this.getRequestID(broadcastResponse.txhash)
+  }
+
+  async getRequestID(txHash) {
```
Comment (prin-r) [inline helpers/bandchain.js/src/lib/BandChain.js]: I saw retryTimeout = 200 at "getRequestProof". So should we add it here as well ?

**Review by taobun — CHANGES_REQUESTED** [review]
Nope

File: helpers/README.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# BandChain Helpers
+
+A collection of tools that simplify interactions with BandChain. Note that this is not a part of BandChain standard and subject to change more often.
+
+## Supported Programming Languages
+
+| Language                        | Status         |
+| ------------------------------- | -------------- |
+| [`Python 3 🐍`](pyobi)          | 🤔 Not Started |
```
Comment (taobun) [inline helpers/README.md]: obi LUL

**Review by taobun — APPROVED** [review]

**Pull request merged**